app/views/admin/api_docs/base/index.html.slim

```diff
@@ -6,9 +6,22 @@
       span.SwaggerUpgradeBox-logo
       | ActiveDocs now supports Swagger 2.0. Read here why it makes sense to upgrade your service specs to Swagger 2.0
 
+- @free_floating_api_docs_services = [1,2]
 - if @api_docs_services.any?
   = render partial: 'active_docs_table', locals: {service: @service, api_docs_services: @api_docs_services}
-- else
+- elsif @free_floating_api_docs_services.any?
```

Comment:
Although this solution that I am going to say is not very elegant, it works so you can use it for testing if you want: `- elsif service.account.api_docs_services.where(service_id: nil).any?`

But I will do something decent instead 👍

Reply:
well, decent enough for me but I'm known for being indecent…

Reply:
The problem of this is I am killing the _Law of Demeter_ by faaaar, to the point that it is not even understandable over time. From the view I am calling to the service, fine, then its account, fine too, then its api_docs_services, and inside it, it even checks that its service_id is nil (too much implementation detail of the DB for the view).

Reply:
Yes but at the same time very readable…

Reply:
This seems very readable to me too but without these problems, no?
![image](https://user-images.githubusercontent.com/11318903/48839524-11860400-ed8c-11e8-9bec-4867378b6c12.png)

Reply:
Sorry, the view is mistaken, I mean: `- elsif current_account.api_docs_services_without_service.any?`